container-images/cann/Containerfile

```diff
@@ -0,0 +1,35 @@
+# Base image with CANN for compilation
+ARG ASCEND_VERSION=8.0.rc2.beta1-910b-openeuler22.03-py3.10
+
+FROM quay.io/ascend/cann:${ASCEND_VERSION} AS builder
+ARG GOLANG_VERSION
+COPY ../scripts /scripts
+RUN export http_proxy=http://172.17.0.1:7890 && https_proxy=http://172.17.0.1:7890 && chmod +x /scripts/*.sh && \
```

Comment:
Will need to remove the hardcoded proxy strings before merge. I would recommend putting this whole section in:

```
container-images/scripts/build_llama_and_whisper.sh
```

as "cann" specific scripting.